f5_openstack_agent/lbaasv2/drivers/bigip/icontrol_driver.py

```diff
@@ -389,6 +390,11 @@ def __init__(self, conf, registerOpts=True):
         self.pool_manager = resource_helper.BigIPResourceHelper(
             resource_helper.ResourceType.pool)
 
+        if self.conf.password_cipher_mode:
```

Comment:
this won't work.   should be encrypt_password?

Reply:
Sorry, not saved before checkin. Change them both to password_cipher_mode.